plugin/src/main/scala/ch/epfl/scala/ProfilingPlugin.scala

```diff
@@ -294,6 +295,16 @@ class ProfilingPlugin(val global: Global) extends Plugin {
         override def run(): Unit = {
           super.run()
 
+          if (!SettingsOps.areStatisticsEnabled(global)) {
+            val flagName = global.settings.Ystatistics.name.replace("-V", "-Y")
```

Comment:
normally if `-V` is supported, it is canonical, and normally if a `-Y ` form exists as well, it is only a deprecated alias. So unless you want to argue otherwise, I should we be documenting and recommending the `-V` form?

Reply:
> normally if -V is supported, it is canonical, and normally if a -Y  form exists as well, it is only a deprecated alias.

Thanks! Is this Scala-version-specific logic?

Reply:
```
scala> settings.Ystatistics.name
val res1: String = -Vstatistics

scala> settings.Ystatistics.abbreviations
val res2: List[String] = List(-Ystatistics)
```

Reply:
But in 2.12, 

```
scala 2.12.18> settings.Ystatistics.name
res0: String = -Ystatistics

scala 2.12.18> settings.Ystatistics.abbreviations
res1: List[String] = List()
```

Reply:
So just removing `.replace("-V", "-Y")` should give the right flag on all versions

Reply:
Yeah, perhaps this difference between `-Y` and `-V` is too subtle for the end users so we can show `-V` for 2.13 and `-Y` for 2.12. Any objections?

Reply:
SGTM

Reply:
I've submitted https://github.com/scalacenter/scalac-profiling/pull/52 to update the readme accordingly.

In the long run, it would be good to move the documentation in the blog post to some more permanent, updatable location. The old post could then have a note at the top redirecting readers to the current documentation. (This is what we did, for example, with https://www.lightbend.com/blog/scala-inliner-optimizer which eventually turned into https://docs.scala-lang.org/overviews/compiler-options/optimizer.html.)